udata/api/__init__.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, app=None, **kwargs):
         self.authorizations = {
             'apikey': {
                 'type': 'apiKey',
-                'passAs': 'header',
-                'keyname': HEADER_API_KEY
+                'in': 'header',
+                'name': HEADER_API_KEY
```

Comment:
It's only used in Swagger specifications. It wasn't working before (it was expressed for Swagger 1.x). Now it is (used in Swagger API doc only)